independent-publisher-connectors/Finnish BIS/apiDefinition.swagger.json

```diff
@@ -445,8 +445,7 @@
                               "type": "string"
                             },
                             "change": {
-                              "type": "integer",
-                              "format": "int32"
+                              "type": "string"
```

Comment:
@tpertila can you please explain why did you change the type here? Will this affect connector users in any way?

Reply:
The current datatype is wrong, so if the query result contains the change column (in businessIDChanges ), it fails. Because API returns string, not integer.

So after the change users get results instead of an error message when searching companies having that change information on the resultset.